default.json

```diff
@@ -4,8 +4,8 @@
   "extends": [
     ":semanticCommits",
     "config:best-practices",
```

Comment:
Best practices still has github actions pinning as well, maybe we remove this and add just the parts of that we want

```diff
-    "config:best-practices",
+   "config:recommended",
+   ":configMigration",
+   ":pinDevDependencies"
```

Which would remove (which come [from best-practices](https://docs.renovatebot.com/presets-config/#configbest-practices)):
```
    "docker:pinDigests",
    "helpers:pinGitHubActionDigests",
```